src/riscv/lib/src/machine_state.rs

```diff
@@ -536,7 +547,7 @@ impl<MC: memory::MemoryConfig, BCC: BlockCacheConfig, B: Block<MC, M>, M: backen
             }
         }
 
-        ControlFlow::Continue(())
+        ControlFlow::Continue(StepCount(1)) // Default step count increment
```

Comment:
You could implement `Default` for `StepCount` and use `StepCount::default()` everywhere instead.